sdks/python/README.md

```diff
@@ -0,0 +1,90 @@
+# Apache Beam
+
+[Apache Beam](http://beam.apache.org/) is a unified model for defining both batch and streaming data-parallel processing pipelines, as well as a set of language-specific SDKs for constructing pipelines and Runners for executing them on distributed processing backends, including [Apache Flink](http://flink.apache.org/), [Apache Spark](http://spark.apache.org/), [Google Cloud Dataflow](http://cloud.google.com/dataflow/), and [Hazelcast Jet](https://jet.hazelcast.org/).
+
+
+
+## Overview
```

Comment:
https://beam.apache.org/documentation/sdks/python/ has some information that can be reused here, I think.

Reply:
Good Point,  I will include the details here.